docs/studies-rmd.md

```diff
@@ -0,0 +1,75 @@
+---
+id: studies-rmd
+title: REMDESIVIR STUDY
+hide_title: true
+hide_table_of_contents: true
+---
+
+##### Top
+
+# Remdesivir
+
+**RESEARCH QUESTION: For patients with new COVID-19 infection, how does the addition of Remdesivir affect the outcomes (1) severe disease (indicated by mechanical ventilation) and (2) inpatient death?**
+
+### Background
+
+Remdesivir are currently under study as possible treatments for COVID-19. Before randomized clinical trials read out, a retrospective observational study can provide insight into the efficacy of this drug.
+
+### Study Design
+
+The study examines the effect of initiation of Remdesivir in hospitalized COVID-19 patients on disease severity. The study involves two study groups (those receiving or not receiving Remdesivir), as depicted below:
+
+![Remdesivir cohort pathways](rmd_cohorts.png)
+
+Four outcomes are considered:
+
+1. Discharge without being placed on mechanical ventilation during stay
+2. Discharge after invasive mechanical ventilation (IMV) during stay
+3. Inpatient death without being placed on mechanical ventilation during stay
+4. Inpatient death after invasive mechanical ventilation (IMV) during stay
+
+These outcomes correspond to increasing levels disease severity. The study uses mechanical ventilation as a proxy for severe but non-fatal disease. The outcome is retrospective over the entire patient visit and capture the worst severity of illness during the visit. For example:
+
+* Patient is put on a ventilator (level 2) and then succumbs to the disease (level 3); the outcome is level 3 (death).
+* Patient is temporarily put on a ventilator (level 2), then taken off the ventilator (level 1) and discharged; the outcome level 2.
+
+### Study Populations
+
+**Study Period**: January 1, 2020 - Present
+
+**Index date**: `COVID-19-positive-date`
+
+* **Entry Cohort**:
+  * Age at [COVID-19 Positive Date](/docs/cohorts#covid-19-positive-date) ≥ 18 years AND
+  * [COVID-19 Positive Date](/docs/cohorts#covid-19-positive-date) after Jan 1, 2020 AND
+  * [COVID-19 Related Hospitalization](/docs/cohorts#covid-19-related-hospitalization)
+* **Target Cohort**:
+  * Entry Cohort AND
+  * [Inpatient Remdesivir Use](/docs/cohorts#inpatient-remdesivir-use) AND NOT [Prior Remdesivir Use](/docs/cohorts#prior-remdesivir-use) [1]
+* **Control Cohort**:
+  * Entry Cohort AND
+  * NOT [Inpatient Remdesivir use](/docs/cohorts#inpatient-remdesivir-use) AND NOT [Prior Remdesivir Use](/docs/cohorts#prior-remdesivir-use)
+
+### Outcome Definitions
```

Comment:
In the other studies we're just pointing to the outcome cohort pages, instead of defining them here.

Reply:
It looks consistent to me.